ui/src/app/components/copy-modal.tsx

```diff
@@ -112,14 +117,20 @@ const NotebookCdrMismatch = (props: {text: string}) =>
       </FlexRow>
     </div>;
 
-const ConceptSetRestrictionText = () => <div style={styles.conceptSetsRestriction}>
+const ConceptSetRestrictionText = () => <div style={styles.restriction}>
   Concept sets can only be copied to workspaces using the same CDR version.
```

Comment:
may want to add "access tier" here too.  It's implicit in this statement but that may not be obvious to users.